### ISSUE TITLE
[ZOOM]: GIBCT® Colmery 107 - Table of locations scrolls horizontally and is a bit difficult to read on iOS

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#1658]
## Description

The locations table becomes somewhat difficult to read at mobile size. The attached screenshot was captured on an iOS device during some mobile screen reader testing.

/cc @emilywaggoner for design visibility.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Acceptance Criteria

* As an iOS / Android mobile user, I want to be able to read the table of locations as easily as possible.

## Environment

* iOS 12 latest
* iPhone 8
* Safari
* https://staging.va.gov/gi-bill-comparison-tool/profile/11806124

## Possible Fixes (optional)

UPDATE: The previous solution has some [accessibility challenges](https://developer.paciellogroup.com/blog/2018/03/short-note-on-what-css-display-properties-do-to-table-semantics/) and is not recommended. 

After talking to @amyknox , it was determined the team would use the same approach as the Historical Information table further down the same page. This table is shown on wider screens, and hidden with `display: none;` in favor of a list on smaller screens. We agreed this was the best way forward with respect to prior art and more research needed on a responsive table solution. 

### Desired solution:
All school names should be bold including branch and extension campuses. 
We don't need to pre-pend "School Name:" in front of the school name nor "Location:" in front of the address. We do want "Estimated housing:" in front of the housing allowance amount.

![Screen Shot 2019-09-11 at 2 13 39 PM](https://user-images.githubusercontent.com/41790951/64723430-8272e580-d49e-11e9-86ec-7df826547346.png)

Screenshots attached of existing behavior, with notes.

~I found a possible solution that uses `data-` attributes and CSS to switch the table to list-like displays at small screen sizes: https://blog.easy-designs.net/archives/responsive-tables/~

~This would need to be tested with screen readers, but could be a pattern for wider adoption.~

## PR
https://github.com/department-of-veterans-affairs/vets-api/pull/3268

## WCAG or Vendor Guidance (optional)

* [Understanding Reflow](https://w3c.github.io/wcag21/understanding/21/reflow.html)

## Screenshots or Trace Logs

![unnamed.jpg](https://images.zenhubusercontent.com/5ac217b74b5806bc2bcd3fc8/f90e2635-e83a-401c-8c8a-7d37cfa96404)

---
**Existing table at wider screen width**

![Screen Shot 2019-09-10 at 9 58 06 AM](https://user-images.githubusercontent.com/934879/64626504-8b3abd00-d3b3-11e9-95fb-493482e0132f.png)

---
**Mobile-width behavior**

![Screen Shot 2019-09-10 at 9 58 15 AM](https://user-images.githubusercontent.com/934879/64626550-9b529c80-d3b3-11e9-89d0-17962d51f7a6.png)

**Comment by amyknox** [comment]
@dmccurdybah For the mobile friendly (list) view of School locations data...

We don't need to pre-pend "**School Name:**" in front of the school name nor "**Location:**" in front of the address.  Final version should look a lot like the attached screenshot.


![image](https://user-images.githubusercontent.com/41081945/64709221-e936d580-d483-11e9-98b8-016354eb6faa.png)

**Comment by dmccurdybah** [comment]
Can be tested on vets-website branch `bah-school-locations`

**Comment by sonntag-adam** [comment]
Branches
![Screen Shot 2019-09-13 at 7.59.20 AM.png](https://images.zenhubusercontent.com/5c98df9c534e09000148f8d1/51adcc9c-6773-4038-b50b-cbf4c2522422)
##
Extensions
![Screen Shot 2019-09-13 at 8.00.16 AM.png](https://images.zenhubusercontent.com/5c98df9c534e09000148f8d1/a2975ebc-238b-420d-b279-0869db912344)
##
Branches and Extensions
![Screen Shot 2019-09-13 at 8.02.09 AM.png](https://images.zenhubusercontent.com/5c98df9c534e09000148f8d1/3d45763d-f180-48a0-81e5-b220549b374d)

**Comment by sonntag-adam** [comment]
@ccruzgra how does this look

**Comment by ccruzgra** [comment]
@sonntag-adam Thanks for posting this. Could you share a screen @ 320px?

**Comment by sonntag-adam** [comment]
320px Branches only
![Screen Shot 2019-09-13 at 9.29.02 AM.png](https://images.zenhubusercontent.com/5c98df9c534e09000148f8d1/e2cf2c62-5bb5-4438-9b76-50409c5e9eb3)
##
320px Extensions only
![Screen Shot 2019-09-13 at 9.29.40 AM.png](https://images.zenhubusercontent.com/5c98df9c534e09000148f8d1/4605c42d-9c5c-4fb1-b615-18e91d892eeb)
##
320px Branches and Extensions
![Screen Shot 2019-09-13 at 9.30.42 AM.png](https://images.zenhubusercontent.com/5c98df9c534e09000148f8d1/4f430991-77f1-436b-83fe-ff791a9f1a83)
![Screen Shot 2019-09-13 at 9.30.49 AM.png](https://images.zenhubusercontent.com/5c98df9c534e09000148f8d1/7b1a9d18-0bfd-4d39-8f93-43e6dae9a392)

**Comment by sonntag-adam** [comment]
@ccruzgra

**Comment by ccruzgra** [comment]
@sonntag-adam This checks out with UXers! Thank you.

**Comment by sonntag-adam** [comment]
Merged into staging. Testing passes QA

**Comment by sonntag-adam** [comment]
@1Copenut changes are in staging.

**Comment by 1Copenut** [comment]
@sonntag-adam I retested this zoomed in and with screen readers (JAWS, NVDA, VoiceOver). Working as expected, moving to close.
